karoo_gp/test/util.py

```diff
@@ -0,0 +1,78 @@
+import hashlib, os, sys
+import numpy as np
+import sklearn.model_selection as skcv
+
+def hasher(msg):
+    if type(msg) != bytes:
+        if type(msg) != str:
+            msg = str(msg)
+        msg = bytes(msg, 'utf-8')
+    hasher = hashlib.md5()
+    hasher.update(msg)
+    return hasher.hexdigest()
+
+def load_data(kernel, save_dir=''):
+    '''Returns pre-loaded datasets & fields; Adapted from fx_data_load'''
+    karoo_dir = 'karoo_gp'  #os.path.dirname(os.path.realpath(__file__))
+    filename = {
+        'c': karoo_dir + '/files/data_CLASSIFY.csv',
+        'r': karoo_dir + '/files/data_REGRESS.csv',
+        'm': karoo_dir + '/files/data_MATCH.csv',
+        'p': karoo_dir + '/files/data_PLAY.csv',
+    }[kernel]
```

Comment:
It might be better to define this dict as a global object in `base_class.py` and import it instead of duplicating it.

Reply:
Save a pointer to Base_GP in every subclass

Reply:
This is now accessible through the `paths` fixture, but since this is not a fixture nor a test function, you will have to pass the path object down to this function from the caller.